app/services/chartService.js

```diff
@@ -51,23 +33,22 @@ angular.module('portainer.services')
             {
               ticks: {
                 beginAtZero: true,
-                callback: percentageBasedAxisLabel
+                callback: ScalesCallback
               }
             }
           ]
         }
-      }
-    });
-  };
+      };
+  }
 
-  service.CreateMemoryChart = function(context) {
+  function CreateChart (context, Label, TooltipCallback, ScalesCallback ) {
```

Comment:
Please use Camel Case with lowercase first letter for function parameters. Also no need for an extra space before the final `)`.